universal-login-relayer/test/relayer/routes/wallet.js

```diff
@@ -70,11 +70,17 @@ describe('E2E: Relayer - WalletContract routes', async () => {
       };
       const balanceBefore = await provider.getBalance(otherWallet.address);
       const signedMessage = await createSignedMessage(msg, wallet.privateKey);
-      const {status} = await chai.request(relayer.server)
+      const {status, body} = await chai.request(relayer.server)
         .post('/wallet/execution')
         .send(signedMessage);
       expect(status).to.eq(201);
-      expect(await provider.getBalance(otherWallet.address)).to.eq(balanceBefore.add(msg.value));
+      await waitExpect(async () => expect(await provider.getBalance(otherWallet.address)).to.eq(balanceBefore.add(msg.value)));
+      const checkStatusId = async () => {
+        const statusById = await chai.request(relayer.server)
+          .get(`/wallet/execution/status/${body.transaction}`);
+        expect(statusById.body.transactionHash).to.not.be.null;
+      };
+      await waitExpect(() => checkStatusId());
```

Comment:
No need to use ugly waitExpect, instead you can use `stopAndWait()` to make sure queue will be empty before using `expect`.